### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by serjgrm**
- [DEMO LINK](https://serjgrm.github.io/layout_calendar/)
- [TEST REPORT LINK](https://serjgrm.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```
Comment (anastasiiavorobiova) [inline src/index.html]:
```suggestion
    <div class="calendar calendar--month... calendar--.....">
```
Screenshot 2023-12-08 at 16 18 11

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+    &:hover{
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+  }
+}
+
+@for $day from 1 through $month-length {
+  .calendar__day:nth-child(#{$day}) {
+    display: flex;
```
Comment (anastasiiavorobiova) [inline src/styles/main.scss]:
```suggestion

```

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+    &:before {
+      display: block;
+      font-size: 30px;
+      font-family: Arial, sans-serif;
```
Comment (anastasiiavorobiova) [inline src/styles/main.scss]:
```suggestion

```
There is no need to produce common styles for each day, consider using basic class for common CSS

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great start! Keep doing!

File: src/index.html (hunk 1 above)
Comment (serjgrm) [reply src/index.html]: Can u please reply for my questions in MA chats ? I tag u.

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Please implement this logic (https://github.com/mate-academy/layout_calendar/pull/2964/files#r1420532392) if you have questions feel free to ask in chat

**Review by ghost — CHANGES_REQUESTED** [review]
Your demo link is broken :(

**Pull request closed without merging**